securedrop/tests/conftest.py

```diff
@@ -110,37 +107,39 @@ def insecure_scrypt() -> Generator[None, None, None]:
         yield
 
 
-def setup_gpg(gpg_dir: Path) -> str:
```

Comment:
All I did here was move the logic in `setup_gpg()` to `setup_journalist_key_and_gpg_folder()`.